src/app/shared/components/lib/default/default.module.ts

```diff
@@ -0,0 +1,11 @@
+import { NgModule }           from '@angular/core';
+import { CommonModule }  from '@angular/common';
+
+import { DefaultComponent } from './default.component';
+
+@NgModule({
+  imports:      [ CommonModule ],
+  declarations: [ DefaultComponent ]
+})
```

Comment:
Add `exports: [ DefaultComponent]` so this component can be used in the application.